services/core-web/src/components/noticeOfWork/applications/review/activities/UndergroundExploration.js

```diff
@@ -28,11 +32,198 @@ const propTypes = {
   undergroundExplorationTypeOptions: CustomPropTypes.options.isRequired,
   renderOriginalValues: PropTypes.func.isRequired,
   isPreLaunch: PropTypes.bool.isRequired,
+  undergroundExplorationTypeOptionsHash: PropTypes.objectOf(PropTypes.string).isRequired,
 };
 
 export const UndergroundExploration = (props) => {
+  const acceptableUnits = props.unitTypeOptions.filter(
+    ({ value }) => value === "MEC" || value === "MTN"
+  );
   return (
     <div>
+      <Row gutter={16}>
+        <div className="field-title">
+          Proposed Activities
+          {props.isPreLaunch && <NOWFieldOriginTooltip />}
+        </div>
+        <Col sm={24}>
+          <Field
+            id="proposed_bulk_sample"
+            name="proposed_bulk_sample"
+            label={
+              <>
+                Bulk Sample
+                <NOWOriginalValueTooltip
+                  originalValue={
+                    props.renderOriginalValues("underground_exploration.proposed_bulk_sample").value
+                  }
+                  isVisible={
+                    props.renderOriginalValues("underground_exploration.proposed_bulk_sample")
+                      .edited
+                  }
+                  style={{ marginLeft: "5px" }}
+                />
+              </>
+            }
+            type="checkbox"
+            disabled={props.isViewMode}
+            component={RenderCheckbox}
+          />
+        </Col>
+        <Col sm={24}>
+          <Field
+            id="proposed_de_watering"
+            name="proposed_de_watering"
+            label={
+              <>
+                De-watering
+                <NOWOriginalValueTooltip
+                  originalValue={
+                    props.renderOriginalValues("underground_exploration.proposed_de_watering").value
+                  }
+                  isVisible={
+                    props.renderOriginalValues("underground_exploration.proposed_de_watering")
+                      .edited
+                  }
+                  style={{ marginLeft: "5px" }}
+                />
+              </>
+            }
+            type="checkbox"
+            disabled={props.isViewMode}
+            component={RenderCheckbox}
+          />
+        </Col>
+        <Col sm={24}>
+          <Field
+            id="proposed_diamond_drilling"
+            name="proposed_diamond_drilling"
+            label={
+              <>
+                Diamond Drilling
+                <NOWOriginalValueTooltip
+                  originalValue={
+                    props.renderOriginalValues("underground_exploration.proposed_diamond_drilling")
+                      .value
+                  }
+                  isVisible={
+                    props.renderOriginalValues("underground_exploration.proposed_diamond_drilling")
+                      .edited
+                  }
+                  style={{ marginLeft: "5px" }}
```

Comment:
I've noticed this styling in several places, do we want to have a css class for it?

Reply:
yes, good idea. I'll update that

Reply:
Okayy - I switched this to  a class with the same code, but since this is getting passed into a different component the className does not override correctly. So I reverted the changes back to the style

below is where the component reads this prop:
```
<span style={props.style} className="violet">
        Edited
      </span>
      ```